### PR TITLE
Main page

### LINKED ISSUE #1: [1pt]creating main page
On the main page, the user can see a list of items that you selected as a theme.

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello guys 😀 
I completed Homepage, but still no style is added except for main page slider which show Items (flights), I used this document for that : https://react-slick.neostack.com/docs/example/simple-slider
also, this link might change according to the name we select for details page:
https://github.com/ZahraArshia/book-flight-frontend/blob/83005c9f2b7fe662046ad72ffeca58d7e4239ad5/src/components/Homepage.js#L50
please check it and let me know if there is a problem, or if I need to add anything. Thank you 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,17 +6,20 @@
...
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "react-slick": "^0.29.0",
     "redux": "^4.2.0",
     "redux-thunk": "^2.4.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set PORT=4000 && react-scripts start",
```
Comment (nevisende) [inline package.json]: I think you use Windows as OS and that script isn't working on my OS(Ubuntu) If you read and change according to [that page](https://stackoverflow.com/questions/40714583/how-to-specify-a-port-to-run-a-create-react-app-based-project) you can start it for all OS.

**Review by nevisende — DISMISSED** [review]
## Hi @ZahraArshia 

Good job so far! I know you have worked hard on making this milestone There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️
- [x] When I visit the home page I can't see anything, empty. If you don't have that purpose you should check and fix it 
- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (ZahraArshia) [reply package.json]: Oh, you are right. I will change it back to default, and we can change the API port instead.

**Comment by ZahraArshia** [comment]
Hello @nevisende 
thanks for the review, I changed the port back to default. As for empty main page it's because it is based on a promise to get items from API, since item controller is not implemented in the backend yet, the promise stays in the pending state, so it doesn't show anything. If you replace it with an array of object temporary, you can see them on the display.
We can add a loading page to avoid this situation, if the connection to API is not established.

**Review by saied2035 — APPROVED** [review]
Great work you did. 👏 

fvq6JHq

It's time to merge! 🔥

**Pull request merged**